public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -274,6 +293,30 @@ export class SharedPreferences extends PureComponent<Props, State> {
               id="locale-select"
             />
           </Field>
+          {config.featureToggles.localeFormatPreference && (
+            <Field
+              loading={isLoading}
+              disabled={isLoading}
+              label={
+                <Label htmlFor="locale-select">
+                <span className={styles.labelText}>
+                  <Trans i18nKey="shared-preferences.fields.locale-format-label">Region format</Trans>
+                </span>
+                <FeatureBadge featureState={FeatureState.experimental} />
+              </Label>
+              }
+              description={t('shared-preferences.fields.locale-format-description', 'Choose your region to see the corresponding date, time and number format')}
+              data-testid="User preferences locale format drop down"
+            >
+              <Combobox 
+              value={locales.find((loc) => loc.value === locale)?.value || moment.locale()}
```

Comment:
I think we should get rid of this `moment.locale()` here. I think it should default to 'Default' or something like that, similar to Language, if it's not set.

Reply:
Which locale should be the default? The language is English, but the locale? 🤔

Reply:
If you don't have a preference stored, this would default it to `en-US` https://github.com/grafana/grafana/blob/main/pkg/api/index.go#L53

The dropdown should show 'Default' when it does not have a value selected.